### PR TITLE
Terms

### PR DESCRIPTION
**Pull request by Haynar216**
Issue #74

### CODE REVIEW

File: src/components/Terms.vue [hunk 1]
```diff
@@ -6,38 +6,118 @@
...
       </div>
       <br>
+      <div>
+        <h3 align="left"> 1. Parties</h3>
+      </div>
       <div class="txt"
-           align="left" >
-        <p> Version 0.1.0:Active as of 25th May 2018</p>
+           align="justify">
+        <p>  1.1. This agreement is between you ( ‘You’ or ‘Your’ in this agreement) and V SYSTEMS Company (‘V SYSTEMS’ in this agreement). V SYSTEMS Web Wallet Application is a cryptocurrency wallet application baseon V SYSTEMS blockchain. You acknowledged that any use of V SYSTEMS Web Wallet Application  is subject to these Terms of Service.</p>
```
Comment (SheldonYS) [inline src/components/Terms.vue]: "baseon" change to "base on"

**Comment by SheldonYS** [comment]
@sunnyking I suggest change all "virtual currency" to "crypto currency"

File: src/components/Terms.vue [hunk 2]
```diff
@@ -6,38 +6,118 @@
...
+        <p>2.1. V SYSTEMS Web Wallet Application</p>
+        <p>V SYSTEMS has created means for users to access and transfer ownership of VSYS coins (such as bitcoin) under the name "V SYSTEMS Wallet...". By entering into this agreement You will receive a license to use any of the V SYSTEMS Web Wallet Application. You should only use the V SYSTEMS Web Wallet Application if you are familiar with virtual currencies. We highly recommend learning the basics of the specific virtual currency that You plan on using the V SYSTEMS Web Wallet Application before running the application.</p>
+        <p>2.1.1. You Own Your Private Keys</p>
+        <p>V SYSTEMS does not store user’s private keys, backup phrases or passwords ("Private Information"). It is very important that You backup Private Information. If You lose Your Private Information then it will not be possible for V SYSTEMS to recover it for You and You may lose access to Your VSYS coins. If You do not understand the above paragraph then we strictly recommend that You to learn more about virtual currencies before using the V SYSTEMS Web Wallet Application.</p>
```
Comment (SheldonYS) [inline src/components/Terms.vue]: For "V SYSTEMS does not store user’s private keys, backup phrases or passwords ...", remove "backup" here.

File: src/components/Terms.vue [hunk 3]
```diff
@@ -6,38 +6,118 @@
...
+        <h3 align="left">8. Indemnity
+        </h3>
+      </div>
+      <div class="txt"
+           align="justify">
+        <p>You agree to indemnify V SYSTEMS against any claims, damages, losses or legal fees incurred due to Your use (or misuse) of V SYSTEMS.</p>
+        <br>
+      </div>
+      <div class="smalltitle">
+        <h3 align="left">9. Non-Waiver
```
Comment (SheldonYS) [inline src/components/Terms.vue]: So, this is "Waiver of Right", right? Not "Non-Waiver".

**Comment by zl730** [comment]
@SheldonYS Is this file reviewed by legal operation department？？Why developers can change the content of instrument of law？？
If change is needed, the agreement file signed by lawyer should be attached.

**Pull request merged**